src/com/cdd/bao/template/ClipboardSchema.java

```diff
@@ -136,7 +142,7 @@ public static String composeAssignmentTSV(Schema.Assignment assn)
 	
 	// ------------ private methods ------------	
 
-	private static JSONObject formatGroup(Schema.Group group) throws JSONException
+	private static JSONObject formatGroup(Schema.Group group, SchemaVocab schvoc) throws JSONException
```

Comment:
JSONException is a runtime exception and not a checked exception. It therefore doesn't need to be declared.

Reply:
Declaring it forces it to be trapped by the caller, which basically makes it behave like a checked exception, and that's intentional.